### PR TITLE
Salaboy api refactoring 3rd

### PR DESCRIPTION
**Pull request by erdemedeiros**

### CODE REVIEW

File: activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    private KeycloakInstanceWrapper keycloakInstanceWrapper;
+
+    @Autowired
+    public KeycloakUserGroupManager(KeycloakInstanceWrapper keycloakInstanceWrapper) {
+        this.keycloakInstanceWrapper = keycloakInstanceWrapper;
+    }
+
+    @Override
+    public List<String> getUserGroups(String username) {
+        // Verify that the user exist
```
Comment (erdemedeiros) [inline activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java]: Unnecessary comment

File: activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                groups.add(groupRepresentation.getName());
+            }
+        }
+
+        return groups;
+    }
+
+    @Override
+    public List<String> getUserRoles(String username) {
+        // Verify that the user exist
```
Comment (erdemedeiros) [inline activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java]: Unnecessary comment

File: activiti-cloud-services-common-security-keycloak/pom.xml [hunk 3]
```diff
@@ -11,8 +11,10 @@
   <name>Activiti Cloud Services :: Common Keycloak Security</name>
   <dependencies>
     <dependency>
-      <groupId>org.activiti.cloud</groupId>
-      <artifactId>activiti-cloud-services-common-security</artifactId>
+      <groupId>org.activiti</groupId>
+      <artifactId>activiti-spring-security</artifactId>
+      <!-- @TODO: do dep mgmt -->
```
Comment (erdemedeiros) [inline activiti-cloud-services-common-security-keycloak/pom.xml]: To be removed

File: activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java (hunk 1 above)
Comment (Salaboy) [reply activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java]: @erdemedeiros please remove

File: activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java (hunk 2 above)
Comment (Salaboy) [reply activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java]: @erdemedeiros please remove

File: activiti-cloud-services-common-security-keycloak/pom.xml (hunk 3 above)
Comment (Salaboy) [reply activiti-cloud-services-common-security-keycloak/pom.xml]: @erdemedeiros please remove

File: activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java (hunk 1 above)
Comment (erdemedeiros) [reply activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakUserGroupManager.java]: done

**Comment by erdemedeiros** [comment]
This PR is ready to merge.

**Pull request merged**